src/pages/EditMonitor.vue

```diff
@@ -159,12 +162,7 @@
 
                                 <div v-if="remoteBrowsersToggle">
                                     <label for="remote-browser" class="form-label">{{ $t("Remote Browser") }}</label>
-                                    <ActionSelect
-                                        v-model="monitor.remote_browser"
-                                        :options="remoteBrowsersOptions"
-                                        icon="plus"
-                                        :action="() => $refs.remoteBrowserDialog.show()"
-                                    />
+                                    <ActionSelect v-model="monitor.remote_browser" :options="remoteBrowsersOptions" icon="plus" :action="() => $refs.remoteBrowserDialog.show()" />
```

Comment:
I've spent a few hours trying to sort out the diff here. It looks like the changes are twofold: the auto-formatter I used in my environment shuffled things around, and I did not pay attention when I committed, and brought these changes in.

It does not appear there is an easy or straightforward way to "revert" the auto-formatting changes... Three options, you decide:
1) I can do is grab the latest version of the file from main branch and paste the lines of code I changed from the most recent commit to this file, and re-commit.
2) We make an educated assumption that most of the unrelated changes were due to my autoformatter, and I run a proper ES Lint and re-commit, understanding that this may appear to make other unrelated changes to the file.
3) We could nuke this PR and I could start again from scratch, copying and pasting the code I have already worked on.

Reply:
Here is the file through proper ES Lint: 4699a1ccd80a53d3ec9d612a5b8fde43107295d2

Reply:
This is better, but still very likely that things are going to get missed.
Could you copy paste the version from master and then (in the git diff) undo reverting the changes you intended to make (=> keeping only what you intended to publish)

Reply:
Resolved by d83c2b90c9d8b82b5c994d34ee2de71233e237c9.